include/interfaces/scalar_reaction_diffusion_convection.h

```diff
@@ -75,16 +75,18 @@ energies_and_residuals(const typename DoFHandler<dim,spacedim>::active_cell_iter
                        FEValuesCache<dim,spacedim> &fe_cache,
                        std::vector<EnergyType> &,
                        std::vector<std::vector<ResidualType> > &residuals,
-                       bool compute_only_system_terms) const
+                       bool ) const
 {
   const FEValuesExtractors::Scalar concentration(0);
 
+  ResidualType alpha = this->alpha;
```

Comment:
scrivo qui perché mi era venuto in mente ma poi passato :) la funzione `reinit`, chiamata sotto, chiama al suo interno `fix_solution_dot_derivative` e usa come alpha quello che viene passato. Pensavo di spostare questo all'interno della funzione reinit. così l'utente può mettere quello che vuole nel dummy number (che in questo caso dummy non è)